src/Command/Command.php

```diff
@@ -29,7 +30,7 @@
  * Includes traits that integrate logging
  * and ORM models to console commands.
  */
-#[\AllowDynamicProperties]
+#[AllowDynamicProperties]
```

Comment:
Why not keeping fcqn?

Reply:
Why does Command still need the annotation in 5.x?

Reply:
Phocs throws an errors about fully qualified names.

Reply:
That is weird, we do not have this in other places.

Reply:
We have had with each attribute and had to change it when merging into 5.x.

All other fully qualified names were cleaned up.

Reply:
> Why does Command still need the annotation in 5.x?

Probably doesn't. Was just trying to fix this constant error we have.